ac/ac-chat/src/Chat.jsx

```diff
@@ -106,20 +110,31 @@ class ChatController extends React.Component<StyledPropsT> {
       <div className={classes.root}>
         <h4 className={classes.header}>{activityData.config.title}</h4>
         <div className={classes.content} ref={node => (this.node = node)}>
-          {data.map(chatmsg => (
-            <Chatmsg msg={chatmsg} key={chatmsg.id} classes={classes} />
+          {values(data).map(chatmsg => (
```

Comment:
The ordering works well, but it might be that if we change the function to generate uuid this will break. I'd suggest adding a field in these LI/messages objects to be sorted on. Timestamp might not work if computed by clients with unsynced clocks, but I think `order = 1 + Object.keys(data).length` can do, if two messages are sent at the same time, they will have the same order and it will not mess the sorting of messages anyway.